Sources/SwiftGraphQLClient/Client/Core.swift

```diff
@@ -255,6 +255,15 @@ public class Client: GraphQLClient, ObservableObject {
         
         return self.execute(operation: operation)
     }
+
+    /// Executes a query request with given execution parameters.
+    public func query(
```

Comment:
I haven't tested those two methods (`query` and `mutate`) yet. I'm not sure we need to test them honestly 🤔

Reply:
I agree 🙂